c10/core/TensorImpl.cpp

```diff
@@ -635,7 +635,9 @@ bool TensorImpl::is_non_overlapping_and_dense_custom() const {
 }
 
 IntArrayRef TensorImpl::sizes_custom() const {
-  if (C10_UNLIKELY(matches_python_custom(SizesStridesPolicy::CustomSizes))) {
+  if (C10_UNLIKELY(
```

Comment:
I think one consequence of this is that wrapper tensor subclasses during tracing (like DTensor) will see calls to ‘aten.size’ in their __torch_dispatch__, and need to handle it properly

Reply:
only if there are symbolic shapes!